charts/stable/bookstack/README_CHANGELOG.md.gotmpl

```diff
@@ -23,5 +31,6 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - N/A
 
-[1.0.0]: #1.0.0
+[2.0.0]: #200
```

Comment:
Why remove the dot notation in the links? I've seen that in a few places, but don't yet understand the purpose behind it :)

Reply:
It's because the section links don't have periods in the URL. [6.0.0](https://github.com/k8s-at-home/charts/tree/master/charts/stable/transmission#6.0.0) versus [600](https://github.com/k8s-at-home/charts/tree/master/charts/stable/transmission#600)